src/main/kotlin/no/nav/personbruker/dittnav/api/config/JsonConfig.kt

```diff
@@ -1,19 +1,14 @@
 package no.nav.personbruker.dittnav.api.config
 
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.module.kotlin.registerKotlinModule
-import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
+import kotlinx.serialization.json.Json
 
-fun buildJsonSerializer(): JacksonSerializer {
-    return JacksonSerializer {
-        enableDittNavJsonConfig()
-    }
+fun buildJsonSerializer(): KotlinxSerializer {
+    return KotlinxSerializer(json())
 }
 
-fun ObjectMapper.enableDittNavJsonConfig() {
-    registerKotlinModule()
-    registerModule(JavaTimeModule())
-    disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+fun json(): Json {
+    return Json {
+        ignoreUnknownKeys = true
```

Comment:
Hva tenker dere om denne? Tidligere gjaldt dette bare for InnloggingsstatusResponse. Det viste seg veldig vanskelig, hvertfall for meg, å få det til å gjelde for bare èn klasse i Kotlinx :)

Reply:
Litt synd å sette dette globalt, hvis det kun gjelder en klasse. Kanskje @chris-santa har noen tips her?